locales/de.json

```diff
@@ -20,10 +20,10 @@
 	"driver-select-vehicle-type.motorbike": "🏍 Roller / Motorrad",
 	"driver-select-vehicle-type.car": "🚗 Auto",
 	"driver-select-vehicle-type.error_only_known_type": "Unbekannte Auswahl, bitte wähle Auto oder Roller / Motorrad aus.",
-	"driver-explain-checkins.text": "Fast fertig, noch ein paar Dinge zur Beachtung. Telegramm hat keine Echtzeit-GPS-Standortunterstützung. Also jedes Mal, wenn du deinen Standort ändert, checke ein. Oder checke jede Stunde ein, wenn du bereit bist neue Aufträge zu akzeptieren. Das wird uns helfen, Passagiere im Umkreis zu finden. Tippe \"Weiter\".",
+	"driver-explain-checkins.text": "Fast fertig, bevor es losgeht gibt es noch ein paar Dinge zu beachten. Telegramm hat keine Echtzeit-GPS-Standortunterstützung. Also jedes Mal, wenn du deinen Standort änderst, checke ein. Oder checke jede Stunde ein, wenn du bereit bist neue Aufträge zu akzeptieren. Das wird uns helfen, Passagiere im Umkreis zu finden. Tippe \"Weiter\".",
 	"driver-explain-checkins.next": "Weiter",
 	"driver-request-location.provide_location": "Bitte teile deinen Standort, damit Passagiere im Umkreis dich finden können.",
```

Comment:
"dich" is at the wrong position in the sentence, I feel like this is a lot better:
"damit Passagiere im Umkreis dich finden können" -> "damit dich Passagiere aus dem Umkreis finden können"